### PR TITLE
Add auto-compaction doc

### PR DESCRIPTION
**Pull request by ganmacs**
ref https://github.com/fluent/fluentd/issues/1126, https://github.com/fluent/fluentd/pull/2805

### CODE REVIEW

File: plugins/input/tail.md [hunk 1]
```diff
@@ -261,6 +261,21 @@ files with dynamic path setting. I will fix this problem in the future.
 Check [this issue](https://github.com/fluent/fluentd/issues/1126).
 
 
+### pos\_file\_compaction\_interval
+
+| type   | default | version |
+|:-------|:--------|:--------|
+| time   | nil     | 1.9.1   |
```
Comment (ganmacs) [inline plugins/input/tail.md]: @repeatedly what is the next version? 1.9.2 or 1.10.0 ?

Comment (ganmacs) [reply plugins/input/tail.md]: @repeatedly ?

Comment (repeatedly) [reply plugins/input/tail.md]: 1.9.2

Comment (ganmacs) [reply plugins/input/tail.md]: Thank you! fixed https://github.com/fluent/fluentd-docs-gitbook/pull/146/files#diff-bf392d7ffee3a4cc8dc25fc615613797R268

**Comment by ganmacs** [comment]
Thank you!

File: plugins/input/tail.md [hunk 2]
```diff
@@ -261,6 +261,21 @@ files with dynamic path setting. I will fix this problem in the future.
...
+|:-------|:--------|:--------|
+| time   | nil     | 1.9.2   |
+
+The interval of doing compaction of pos file.
+The targets of compaction are unwatch file's line, unparsable file's line and duplicated line.
+You can use this value when pos\_file option is set.
+
+```
+pos_file /var/log/td-agent/tmp/access.log.pos
+pos_file_compaction_interval 72h
```
Comment (juliantaylor) [inline plugins/input/tail.md]: the hour unit appears to not be used and it expects seconds, see https://github.com/fluent/fluentd/issues/2917

**Pull request merged**